pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html

```diff
@@ -594,7 +594,7 @@ <h3 class="modal-title">Verify LDAP Configuration</h3>
                                             <clr-icon aria-live="assertive" a11yTooltipTrigger shape="info-circle" size="22">
                                             </clr-icon>
                                             <clr-tooltip-content clrPosition="top-right" clrSize="lg" *clrIfOpen>
-                                                <span>A user name used to test LDAP user search functionality.</span>
+                                                <span>A user name used to test LDAP user search functionality; can only be searched by CN.</span>
```

Comment:
I think in this case we can simplify to:
Enter the CN attribute for the user to test LDAP user search functionality.

Below on line 601, let's change the placeholder to CN